docs/authentication_interoperability.md

```diff
@@ -0,0 +1,160 @@
+
+## Authentication Provider setup and guidelines
+Every Integrator is required to provide an Authentication Provider that will be used to verify the authenticity of the payment request. This is done by
+providing an encrypted data object in any enrolment/payment request that contains data generated by the Authentication Provider.
+
+### Authentication Provider requirements
+In order to facilitate an authenticated payment the Authentication Provider must provide a signed JWS token that is verified by EPP.
+This JWS needs to contain a `PermissionGrant` object that is encoded in the JWS token. It must have
+the structure as seen in the [components overiew](assets/swagger/swagger_epp_components.md).
+
+We recommend using a robust signing algorithm such as `RS256`.
+
+The `Digest` field in the `PermissionGrant` object is a `Base64` encoded SHA-256 hash of the following datapoints.
+
+For enrolment:
+````
+nonce: Must be the same as the corresponding nonce in the PermissionGrant object.
+accountNumber: The Account Number of the enrolment session. 
+tokenRequestorName: The Token Requestor Name.
```

Comment:
While skimming through it again, it occurred to me that it might be helpful to point out once that this is the name registered during the set-up (could refer to Checklist for information exchange in getting_started.md). Obvious to us, but might not be so clear to a pair of new eyes.

Reply:
Great idea, I think that's the pattern we should follow for all of these terms. I'll fix accordingly.